### ISSUE TITLE
reply to a comment

### ISSUE THREAD

**Issue by chitgoks** [highkite/pdfAnnotate#3]
hi. is this the function 

createPopupAnnotation(...)

to create replies to a comment?

**Comment by highkite** [comment]
No. Right now the library has no support for replying to comments. Do you have a PDF file that contains such replies or where it is specified in the PDF standard? I think it is an interesting feature for the library.

**Comment by chitgoks** [comment]
the reply is the same as creating a sticky note. but the difference is
there is an IRT property pointing to the annotation where its comment
serves as the root comment.

hence, this sticky note then beomes the reply to that annotation.

i am no expert when it comes to supplying values in streams since that
requires in depth knowledge.

but i can help give tips if its ok with you.

i have worked on using libraries like itext aspose and pdfbox. but there
will be users who want a javascript solution.

so far yours is the best and easy to use.

there are also some nice things to add in your current code.

like adding paramter to opacity to circle square, line thickness. right now
theyre set to 1.

you can email me @ gmail.

**Comment by LibertyNLP** [comment]
Hi , I wish my work can help you.
I have figured out a solution of pdf.js annotation, use only pdf.js dist, and it is very easy to import into your project.

Demo and source code: https://demos.libertynlp.com/#/pdfjs-annotation
Demo video:https://www.youtube.com/watch?v=IYD7WKzx-yc&t=3s

functions:
① highlight : highlight selected text;
② underline : underline selected text;
③ edit: edit annotation ( copy/delete/add comment/navigate );
④ output annotations : output annotations for importing;
⑤ import annotations : import annotations for current file;
⑥ download annotations : download annotations and comments text;
⑦ undo annotations : undo annotations in current page;
⑧ help : open help file;
